spectator-nflx-plugin/src/main/java/com/netflix/spectator/nflx/SpectatorModule.java

```diff
@@ -110,15 +111,20 @@
     @Inject(optional = true)
     private Config config;
 
+    private Config atlasConfig;
+
     @Inject(optional = true)
     private Clock clock;
 
     Config config() {
-      if (config == null) {
-        LOGGER.warn("no archaius2 binding found, using empty configuration");
-        config = EmptyConfig.INSTANCE;
+      if (atlasConfig == null) {
+        if (config == null) {
+          LOGGER.warn("no archaius2 binding found");
```

Comment:
It looks like this will already get logged as part of `createConfig`. Do we need both?